string key
string metadata
string value
---
bool success